### DISCUSSION TITLE
custom disabled range

### DISCUSSION THREAD

**Discussion by dongvng** [gpbl/react-day-picker#1465] [Support]
are we having range disabled custom style? i cannot seem to find related doc for this.

**Comment by dimitur2204** [comment]
I do not quite get what you mean by range disabled custom style, but this is how you can disable a range and it styles is as regular disabled days https://react-day-picker.js.org/basics/modifiers#disabling-days
``` 
<DayPicker
      mode="range"
      disabled={[
        { before: minDate } as DateBefore,
        { after: maxDate } as DateAfter,
      ]} />
```
